SmartDeviceLinkTests/DevAPISpecs/NSMutableDictionary+StoreSpec.m

```diff
@@ -0,0 +1,165 @@
+#import <Quick/Quick.h>
+#import <Nimble/Nimble.h>
+
+#import "NSMutableDictionary+Store.h"
+
+@interface TestObject : NSObject
+@property (nonatomic, strong, nullable) NSDictionary *json;
+- (instancetype)initWithDictionary:(NSDictionary *)json;
+@end
+
+@implementation TestObject
+- (instancetype)initWithDictionary:(NSDictionary *)json {
+    self = [super init];
+    if (self) {
+        self.json = json;
+    }
+    return self;
+}
+- (BOOL)isEqual:(id)object {
+    if (![object isKindOfClass:[TestObject class]]) {
+        return false;
+    }
+    TestObject *anotherObject = (TestObject *)object;
+    return [self.json isEqual:anotherObject.json];
+}
+@end
+
+QuickSpecBegin(NSMutableDictionary_StoreSpec)
```

Comment:
I would recommend using an actual RPC class here and testing the fail cases as you do below. If you choose not to do that and stick with your current approach, that is okay as well. I am commenting below in case you choose to stick with your current approach.

Reply:
I've sticked current approach because it alarms about mistake in store code and avoid fake alarm